From 98c6ad0bf7728d2d8e520093e948c4811e5b00c5 Mon Sep 17 00:00:00 2001
From: Andrea Leardini <andrea.leardini@nethesis.it>
Date: Fri, 13 Sep 2019 10:38:49 +0200
Subject: [PATCH] Lowered level of some log messages

Some log messages that concern email drafts are now printed to log at
debug level instead of error level
---
 src/main/java/com/sonicle/webtop/mail/Service.java | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/main/java/com/sonicle/webtop/mail/Service.java b/src/main/java/com/sonicle/webtop/mail/Service.java
index 0bfe4aa..3e61477 100644
--- a/src/main/java/com/sonicle/webtop/mail/Service.java
+++ b/src/main/java/com/sonicle/webtop/mail/Service.java
@@ -681,17 +681,17 @@ public class Service extends BaseService {
 		while(retry>0) {
 			try {
 				account.deleteByHeaderValue(HEADER_X_WEBTOP_MSGID,""+msgId);
-				if (retry<3) logger.error("Retry deleting automatic draft succeded");
+				if (retry<3) logger.debug("Retry deleting automatic draft succeded");
 				break;
 			} catch(Throwable t) {
-				logger.error("Error deleting automatic draft",t);
+				logger.debug("Error deleting automatic draft",t);
 			}
 			if (--retry >0) {
-				logger.error("Retrying delete of automatic draft after exception");
+				logger.debug("Retrying delete of automatic draft after exception");
 				try { Thread.sleep(1000); } catch(InterruptedException exc) {}
 			}
 		}
-		if (retry==0) logger.error("Delete of automatic draft failed");
+		if (retry==0) logger.error("Delete of automatic draft failed, msgId: " + msgId + ", mail account username: " + account.getUsername());
 	}
 	
 	private MailAccount getAccount(Identity ident) {
@@ -752,7 +752,7 @@ public class Service extends BaseService {
 			msg.addHeaderLine(HEADER_X_WEBTOP_MSGID+": "+msgId);
 			Exception exc = saveMessage(msg, null, fc);
 		} catch(Exception exc) {
-			logger.error("Error on autosave in drafts!",exc);
+			logger.debug("Error on autosave in drafts!",exc);
 		}
 	}	
 	
@@ -4813,7 +4813,7 @@ public class Service extends BaseService {
                 json=new JsonResult(false, exc.getMessage());
 			}
 		} catch (Exception exc) {
-			Service.logger.error("Exception",exc);
+			Service.logger.debug("Exception",exc);
             json=new JsonResult(false, exc.getMessage());
 		}
         json.printTo(out);
@@ -5020,7 +5020,7 @@ public class Service extends BaseService {
 					//InternetAddress.parse(email.replace(',', ' '), false);
 					getInternetAddress(email);
 				} catch (AddressException exc) {
-					Service.logger.error("Exception",exc);
+					Service.logger.debug("Exception",exc);
 					throw new AddressException(lookupResource(MailLocaleKey.ADDRESS_ERROR) + " : " + email);
 				}
 			}
-- 
2.21.0

